Set daily word font-size based on character count

tbarland77 opened the pull request:
Here is my PR for #95. I don't think it's the cleanest way to have done it... Although it does seem to work well for most longer words.

Example before the changes:
![image](https://user-images.githubusercontent.com/15313520/83698560-bca58980-a5cf-11ea-872a-1964b8d370c0.png)

Example with the changes:

![image](https://user-images.githubusercontent.com/15313520/83698598-d515a400-a5cf-11ea-9b55-b37747458cbb.png)

Anyway, let me know what you think!

jayehernandez commented on client/src/views/Word.vue:
```diff
@@ -25,14 +27,24 @@
-    ...mapState(["dailyData", "loading"])
+    ...mapState(["dailyData", "loading"]),
+    calcualteFontSizeForDailyWord() {
+      if (this.dailyData && this.dailyData.word && this.dailyData.word.word && this.dailyData.word.word.length >= 20) {
```
Good use of placing this inside `computed`, maybe a `switch` case would make this more readable?

`switch (this.dailyData.word.word.length)` can work. Then wrap the entire switch statement in an `if (!!this.dailyData.word.word)` which should only evaluate to true if the value is defined.

tbarland77 replied:
I completely forgot you could use !! to simplify things like that. Thanks for the tip! I also replaced the if elses with a switch case as well. I think it's definitely more readable now.

jayehernandez commented:
Just have a comment on the code but good with the approach of changing the size based on length.

tbarland77 commented:
I just wanted to post another quick update that I changed the switch statement a bit after doing some more testing. It looked like it was always going to the default for some reason. So I updated the switch to just true and now it appears to be working again.

jayehernandez commented:
Changes look good! Will be including this in the next update. Thanks for working on this @tbarland77! 🎉

Closes #95
